ground/gcs/src/plugins/welcome/qml/main.qml

```diff
@@ -90,8 +90,8 @@ Rectangle {
 
                 WelcomePageButton {
                     baseIconName: "hitl"
```

Comment:
Breaks it, if it is important I'll have to dig into it, wasn't immediately obvious how it works.